docs/en/00_Getting_Started/01_Installation/04_Other_installation_Options/Mac_OSX_Homebrew.md

```diff
@@ -0,0 +1,112 @@
+# Mac OSX with Homebrew
+
+This topic covers setting up your Mac as a web server and installing SilverStripe.
+
+OSX comes bundled with PHP, but you're stuck with the version and modules it ships with.
+If you run projects on different PHP versions, or care about additional PHP module support
+and other dependencies such as MariaDB, we recommend an installation through [Homebrew](http://brew.sh/).
+
+Check out the [MAC OSX with MAMP](../Mac_OSX) for an alternative installation process
+which packages up the whole environment into a convenient application.
+
+## Requirements
+
+Since we're compiling PHP, some build tooling is required.
+Download and install the free [Apple Xcode development environment](https://developer.apple.com/downloads).
```

Comment:
I believe that `xcode-select --install` works without having to install XCode manually prior to installation?